workflows/templates/sync-template.yaml

```diff
@@ -36,6 +36,8 @@ spec:
             value: "{{inputs.parameters.THOTH_FORCE_SYNC}}"
           - name: "THOTH_LOG_GRAPH_SYNC_JOB"
             value: "DEBUG"
+          - name: "THOTH_SYNC_INSPECTION_ONLY_GRAPH"
+            value: "True"
```

Comment:
```yaml
            value: "1"
```

Reply:
also for `THOTH_SYNC_LOCAL_FILE` is "1"

Reply:
Yes, generally I do cast to int on boolean values in sources so let's be consistent in the project.

```python
bool(int(os.getenv("THOTH_SYNC_LOCAL_FILE", 0)))
```

Python's booleans are ints in fact (so these operations are much more effective than string comparison and less error-prone):

```python
>>> isinstance(False, int)
True
```

Reply:
Thanks I ll update it :)